p2719.md

```diff
@@ -848,3 +951,56 @@ This proposal does not have any impact on the library, since this only tweaks th
 compiler when it evaluates a new-expression and a delete-expression. In particular, we do not propose adding new
 type-aware free function `operator new` variants in the standard library at this time, althought this could be
 investigated in the future.
+
+## Interactions with coroutines
+
+Coroutines currently permit user defined allocators, but do not have a mechanism to communicate the
+type being allocated. For P2719 we are not adjusting the semantics of coroutine allocations to
+resolve type aware new or delete. The coroutine specification will need to be updated in the future to
+determine what type will be passed, and to include that type in the allocation function resolution
+(similar to what is achievable through `decltype(lambda)`).
```

Comment:
```suggestion
Coroutines currently allow using a custom `operator new` and `operator delete` for allocating the coroutine frame. That is done by looking up in the coroutine's promise type for `Promise::operator new`. However, there is no mechanism to communicate the type being allocated, since that type is only something known by the compiler (and fairly late during translation). This paper does not propose changing how allocation for coroutine frames is customized. In the future, the coroutine specification could be updated to make the type being allocated more visible and to use a typed allocation function, but there is significant enough design space to avoid doing that here.
```